src/planscape/boundary/models.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.gis.db import models
+
+
+class Boundary(models.Model):
```

Comment:
would it make sense to name this FooCollection and name the child table Foo?

Reply:
This does keep it consistent with the Python types in boundary_types.py, but admittedly, there's no analogue there of the "nested" details.